questions 1-14

```diff
@@ -0,0 +1,96 @@
+1. models.py contains classes and fields where tables represent tables and fields represents the column types.
+   Some field options are null,  blank, default, editable unique and more.
+   models.py is mostly incharge of containing the DBs build state.
+
+2. Views is incharge of the logic in the backend. That's where all of the functionality is done for the clients requests.
+   The functions in the view can retrieve data from the db, send an email and do lots of more backend related logic.
+
+3. The url.py file holds all of the URLs that our project works with.
+   When a client enters a url it checks the url.py and if it exists it then will send the request to a view function
+   that will handle the clients request.
+
+4. Migration is the process of applying changes to your model (db schema). Its incharge of keeping your model and db in sync.
+   To add a new migration you need to write into the cmd line: manage.py migrate.
+
+5. The serializer converts complex data types like django model instances into python objects so that we can then
+   convert it into json. The reason we do this is because some things are to complex to immediately convert into json format
+
+6. The django framework is a open-source python web framework that follows the MVC architecture. Its used as a backend tool
```

Comment:
That's true, but the question was about Django **Rest** Framework, which is an addition to Django.
It's basically an add-on to Django to make it easier to implement REST APIs. Please read about it a bit.